synapse/handlers/auth.py

```diff
@@ -1851,15 +1860,24 @@ def verify_short_term_login_token(self, token: str) -> LoginTokenAttributes:
         user_id = get_value_from_macaroon(macaroon, "user_id")
         auth_provider_id = get_value_from_macaroon(macaroon, "auth_provider_id")
 
+        oidc_sid: Optional[str] = None
+        try:
+            oidc_sid = get_value_from_macaroon(macaroon, "oidc_sid")
+        except MacaroonVerificationFailedException:
```

Comment:
This seems to be undefined -- missing import probably from splitting the PR?

Reply:
Yep, definitely. Not around my computer anymore, so feel free to push a fix on my branch, else I'll do that tomorrow

Reply:
Fixed